doc/migrate-to-pymongo4.rst

```diff
@@ -174,6 +174,14 @@ can be changed to this::
 
     names = client.list_database_names()
 
+``directConnection`` defaults to False
```

Comment:
Can you move this to the top of the MongoClient section? That's where the other option changes are documented.

Reply:
Done.